actions.js

```diff
@@ -0,0 +1,38 @@
+export function getActionDefinitions(self) {
```

Comment:
Overall, I do not see the value of this module over just using the generic TCP module.

Reply:
This module is based on my PowerPoint and video software through MPC. The actions part allows me to use 2 or more computers in a network and open the software simultaneously.

but I changed the rest as you requested.
however, when I eliminate the part you mentioned from the package. my system gives an error within the companion.

Reply:
Companion allows you to use 2 or more computers as well. In Companion, you just need to add multiple connections that are of the type Generic: TCP and UDP Request and send the appropriate IP and Port.

Reply:
I understand, but I already created software to receive the commands and I created this code to send the commands that my software works. I use generico, I need to make my own command from my software And also explain to the people who work with me how to do this. And this pptvideo already has the exact commands. I also created a timer to receive commands from the companion and I'm waiting for a companion to be created for it.

Reply:
Your software still needs to have the commands available so that it knows what to do when you send it the commands, but the Generic TCP and UDP Request module will allow you to send the commands to your software.  You can easily tell the event producers that are using your software how to use the Generic TCP and UDP Request module to send commands to your software.

Reply:
This pptvideo module sends ready commands to activate, advance, rewind or close the ppt. But I would like to have this code released, as it is faster to activate the commands. And I don't need to explain how it works to the people who will use it.

Reply:
From the boilerplate text of the Generic TCP module:

> The companion project is designed to make the everyday life of a technician easier. We pefer to have ready made actions, presets and feedbacks for as many products as possible. [...] Do you think your product/device is too insignificant to make a module for? It's probably not.